### PR TITLE
ref #5: revision of the icon-set

### PR DESCRIPTION
**Pull request by ckleis**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no 
| Deprecations? | no
| Fixed issues  | #5 
| License       | MIT

### CODE REVIEW

**Review by umulmrum — CHANGES_REQUESTED** [review]
@ckleis Could you add copyright information for the new images (license + where to find the original images)? Even if the images are free to use, we should be able to trace back where our images originate, even in a few years.

Maybe it makes sense to add another license file in the root directory of the repository, named IMAGE-LICENSE, where we list copyright information for every image (or for a complete directory, if all images have the same origin).

File: IMAGE-LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Image-Licenses
```
Comment (umulmrum) [inline IMAGE-LICENSE]:
```suggestion
Image Licenses
```

File: IMAGE-LICENSE [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+==============
+
+Here you can find the license information about the images used.
```
Comment (umulmrum) [inline IMAGE-LICENSE]:
```suggestion
This file contains license information about the images used in Chameleon System ShopThemeBundle.
```

File: IMAGE-LICENSE [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Sprite-Images:
```
Comment (umulmrum) [inline IMAGE-LICENSE]:
```suggestion
Sprite Images:
```

File: IMAGE-LICENSE [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+--------------
+
+| Sprite Name | Original Name  | Source-URL |
+| ------- | --- | --- |
```
Comment (umulmrum) [inline IMAGE-LICENSE]: The actual licensing information is missing :-)

The license for these images is this one: https://creativecommons.org/licenses/by/3.0/us/legalcode
Could you add another section "License Texts" with a sub-section "CC BY 3.0 US" that contains this link (at the end of the file)? I'd like to paste the complete license text, but unfortunately it isn't available in plain text, so the link has to be enough for now.

Could you also add two further columns to the table?
- One should contain the creator of each image, as given on the respective thenounproject.com pages.
- One should contain the license. All of these images use "CC BY 3.0 US"

**Review by umulmrum — CHANGES_REQUESTED** [review]

File: IMAGE-LICENSE (hunk 1 above)
Comment (ckleis) [reply IMAGE-LICENSE]: done

File: IMAGE-LICENSE (hunk 2 above)
Comment (ckleis) [reply IMAGE-LICENSE]: done

File: IMAGE-LICENSE (hunk 3 above)
Comment (ckleis) [reply IMAGE-LICENSE]: done

File: IMAGE-LICENSE (hunk 4 above)
Comment (ckleis) [reply IMAGE-LICENSE]: done

**Comment by umulmrum** [comment]
Thanks @ckleis

**Pull request merged**